lib/common/common.go

```diff
@@ -471,6 +477,34 @@ func WriteRootfsDir(tarWriter *tar.Writer) error {
 	return tarWriter.WriteHeader(hdr)
 }
 
+func writeStdioSymlinks(tarWriter *tar.Writer, filemap map[string]struct{}, pwl []string) ([]string, error) {
```

Comment:
Can you add some comment here? Just reuse the sentence you put in the commit message.